cot/src/request.rs

```diff
@@ -209,6 +211,24 @@ pub trait RequestExt: private::Sealed {
     #[must_use]
     fn db(&self) -> &Arc<Database>;
 
+    /// Get the email service.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use cot::request::{Request, RequestExt};
+    /// use cot::response::Response;
+    ///
+    /// async fn my_handler(mut request: Request) -> cot::Result<Response> {
+    ///     let email_service = request.email();
+    ///     // ... do something with the email service
+    ///  # unimplemented!()
+    /// }
+    /// ```
+    #[cfg(feature = "email")]
+    #[must_use]
+    fn email(&self) -> &Arc<Email>;
```

Comment:
Same here, this should return `&Email` instead. I know we return `Arc<Database>` just above - that should be fixed, too.